assets/javascripts/overview.js

```diff
@@ -127,4 +130,33 @@ function setupOverview() {
             element.checked = states[element.id.substr(7)];
         });
     }
+
+    var parentChild = document.getElementsByClassName('parents_children');
+    for (var i = 0; i < parentChild.length; i++) {
+        parentChild[i].addEventListener('mouseover', highlightDeps);
+        parentChild[i].addEventListener('mouseout', unhighlightDeps);
+    }
+}
+
+function highlightDeps() {
+    var parentData = JSON.parse(this.getAttribute('parents_deps'));
```

Comment:
This needs to be changed in accordance with the other comment. Besides, it would be more compact to write it directly as `addClassToDependencyJob(this, 'parents_deps', 'highlight_parent');` and `addClassToDependencyJob` takes care of the member access and JSON parsing.

Reply:
I tried to write it directly as `addClassToDependencyJob(this, 'parents_deps', 'highlight_parent');`, but I found I cannot get the `parents_deps` by using `dataset`. such as:
```
function highlightDeps() {
     addClassToDependencyJob(this, 'parents_deps', 'highlight_parent');
}
function addClassToDependencyJob(obj, dataDpes, className) {
    var dataDeps = obj.dataset.dataDeps;
}
```
Do you have any suggestions?  Thanks.